package.json

```diff
@@ -27,7 +27,9 @@
   "license": "MIT",
   "node-red": {
     "nodes": {
-      "ttn": "ttn/ttn.js"
+      "ttn-app": "ttn-app/ttn-app.js",
+      "ttn": "ttn/ttn.js",
```

Comment:
Was thinking this as well, but wanted to keep the scope of the PR smaller. Will add!